OfficeToPDF/PowerpointConverter.cs

```diff
@@ -156,12 +156,14 @@ public static new int Convert(String inputFile, String outputFile, Hashtable opt
                         }
                     }
                     
+                   
                     // Sometimes, presentations can have restrictions on them that block
                     // access to the object model (e.g. fonts containing restrictions).
                     // If we attempt to access the object model and fail, then try a more
                     // sneaky method of getting the presentation - create an empty presentation
                     // and insert the slides from the original file.
                     Fonts fonts = null;
+                    Boolean refFonts = (Boolean)options["powerpoint_ref_fonts"];

```

Comment:
PP: follow the same style like it was used in word (https://github.com/showpad/OfficeToPDF/blob/master/OfficeToPDF/WordConverter.cs#L63)
`Boolean bitmapMissingFonts = !(Boolean)options["powerpoint_ref_fonts"];`